franco/Project/org.fundacionjala.automationproject/src/test/resources/features/searchflight.feature

```diff
@@ -0,0 +1,23 @@
+@SearchFlight
+Feature: SearchFlight
+  Test successfully searching of flight
+
+  Background:
+    Given I navigate to Main Page
+    And I log in to the application
+
+  @SmokeTest
+  Scenario Outline: Search a flight
+    When I select the type of flight as "<Flight-type>"
+    And I choose flight for "<Passengers>" passengers
```

Comment:
I think that the best way is only an And I select the following fields flight "<Passengers>" , location "<Origin>" ,departure date "<Departure-Month>", "<Departure-Day>",return date "<Return-Month>", "<Return-Day>", ......

Reply:
I refactor steps When and And in one.